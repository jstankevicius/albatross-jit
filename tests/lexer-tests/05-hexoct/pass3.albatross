0x0
0xCAFEBA 0xDEAD 0xBEEF 0xABCDEF 0x01234 0x056789
0x00000000ABCDEF
01234567 0 01 02 000000001
473 400 70 3 0
