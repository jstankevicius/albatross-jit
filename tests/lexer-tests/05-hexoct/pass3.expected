1 1 INT 0
1 2 INT 13303482
10 2 INT 57005
17 2 INT 48879
24 2 INT 11259375
33 2 INT 4660
41 2 INT 354185
1 3 INT 11259375
1 4 INT 342391
10 4 INT 0
12 4 INT 1
15 4 INT 2
18 4 INT 1
1 5 INT 473
5 5 INT 400
9 5 INT 70
12 5 INT 3
14 5 INT 0
